daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/dafext.xsd

```diff
@@ -587,15 +587,23 @@
         <xs:restriction base="dfdlx:ParseUnparsePolicyEnum" />
       </xs:simpleType>
       <xs:simpleType>
-        <xs:restriction base="xs:string">
+        <xs:restriction base="xs:token">
           <xs:enumeration value="fromRoot" />
         </xs:restriction>
       </xs:simpleType>
     </xs:union>
   </xs:simpleType>
 
+  <xs:simpleType name="TunableTDMLImplementation">
```

Comment:
One option would be to just manually define it somewhere instead of generating it--maybe in daffodil-tdml-lib which deals with all the implementation stuff now. This simple type isn't actually referenced anywhere now.

Fine with me either way though.